docs/configuration.md

```diff
@@ -218,25 +215,25 @@ when included in queries.
 You should add:
 
 ```yaml
-         - name: unique_things
-           special: unique
+- name: unique_things
+  special: unique
```

Comment:
IMHO we could drop the whole section regarding how long ago 0.8.3 version has been released.
If not we should change "special" to "nativeType" and "unique" to "hyperUnique".